src/patternfly/layouts/Page/styles.scss

```diff
@@ -0,0 +1,73 @@
+// URL.com/guidelines#layout
+@import "../../patternfly-utilities";
+
+$pf-l-page__sidebar-Width:    15rem !default;
```

Comment:
15rem is a magic number, please ask @kybaker if that's the size he wants and if so add a comment explaining why this is here. Maybe this should be a spacer